### PR TITLE
ru.po: update russian translation

### PR DESCRIPTION
**Pull request by DiMan**
1) update ru.po file from new gnucash.pot
2) update some translations
`4669 translated messages, 344 fuzzy translations, 177 untranslated messages`

I will take a look at fuzzy and untranslated messages in the near future.

### CODE REVIEW

File: po/ru.po [hunk 1]
```diff
@@ -12132,10 +12136,9 @@ msgstr "<b>Описание категории</b>"
 
 #: gnucash/gtkbuilder/assistant-hierarchy.glade:341
 msgid "Choose accounts to create"
-msgstr "Выберите счета для создания"
+msgstr "Выбор создаваемых счетов"
```
Comment (fellen) [inline po/ru.po]: Just a note:
You should in the whole program be consistent to use the polite form of address to sb. or not. I don't know the current usage in Russia.

Comment (DiMan) [reply po/ru.po]: Hello Frank,

the old form was simply translated from English 1:1 and sounded like an instruction - not as a short description of wizard step. I have looked at list with all wizard steps and tried to find a better form: as a short description. 

Thanks, DiMang

**Comment by fellen** [comment]
commits  af02f62...5f00259

Please see https://wiki.gnucash.org/wiki/Translation#Special_characters_and_other_tips and https://wiki.gnucash.org/wiki/Translation#Desired for the  mnemonics.

File: po/ru.po [hunk 2]
```diff
@@ -19213,9 +19213,8 @@ msgid "Export the Transactions to a CSV file"
 msgstr "Экспортировать проводки в CSV файл"
 
 #: gnucash/import-export/csv-exp/gnc-plugin-csv-export.c:64
-#, fuzzy
 msgid "Export _Active Register to CSV..."
-msgstr "Экспорт _иерархии счетов в CSV..."
+msgstr "Экспорт текущего журнала в CSV..."
```
[the remark this answers is not in the document]
Comment (DiMan) [reply po/ru.po]: Thanks, I will take a look at this.

**Pull request closed without merging**